navigation/page-index/page-index.js

```diff
@@ -0,0 +1,27 @@
+module.exports = [
+  {
+    title: "Find a docs page",
+    href: "navigation",
+    filePath: "index.md",
+    children: [
+      {
+        title: "Find pages by Cloud Console location",
```

Comment:
```suggestion
        title: "Find by Cloud Console location",
```

Just for parallelism.